src/src/hooks/useRedecedMotion.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+
+const useReducedMotion = (defaultVal = true) => {
+  // Local state to store the reduced motion setting.
+  const [reducedMotion, setReducedMotion] = useState(defaultVal);
+
+  // Callback for media query cahnge events.
+  function queryCnangeHandler(event : any) {
+    // Set the state to the value of the media query.
+    setReducedMotion(event.target.matches);
+  }
+
+  useEffect(() => {
+    // Grab the reduced motion media query,
+    const mediaQuery = window.matchMedia("(prefers-reduced-motion: reduce)");
+    console.log("Reduced Motion", mediaQuery)
```

Comment:
Are we wanting to leave logging in for this?

Reply:
Nope.